project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.0
+sbt.version=1.4.4
```

Comment:
Q: Did 1.4.4 finally fix the memory issue?

Reply:
I'm not familiar with the memory issue. I know upgrading to sbt 1.4.3 was failing on a lot of scala steward PRs, but I didn't really look into it.

Reply:
My main problem is solving the sbt-git-versioning sbt plugin issues.

Reply:
I upgraded to 1.5.0 to fix the remaining issues.